WordPress/twentytwentyfour
# Move a couple patterns to theme.json

**Description**

Closes #299 by shifting a couple patterns to theme.json. This way theme developers have a real-use example to see how they can do the same. 

**Screenshots**

![CleanShot 2023-10-12 at 11 36 10](https://github.com/WordPress/twentytwentyfour/assets/1813435/6f3d6520-6ea0-45cc-b663-b1f4c4415ba0)
![CleanShot 2023-10-12 at 11 56 30](https://github.com/WordPress/twentytwentyfour/assets/1813435/af2c1153-d8ad-4dbc-8c3d-127044344731)



**Testing Instructions**

1. Open up the Patterns inserter.
2. Look in the services category.
3. See the "Three columns of services" pattern available.
4. Look in the testimonial category. 
5. See the "Clients" pattern available. 
6. Make sure the clients pattern is viewed in the `twentytwentyfour/business-about` page pattern.
